lib/fog/kubevirt/compute/models/vm.rb

```diff
@@ -6,6 +6,7 @@ module Kubevirt
     class Compute
       class Vm < Fog::Model
         include VmAction
+        include Shared
```

Comment:
Why is it needed ?

Reply:
`deep_merge!` was missing when I called it from the test. Simple bug fix :)